plugins/covid/loader.py

```diff
@@ -0,0 +1,38 @@
+"""
+Loading Covid data from upstream
+"""
+from elcid.models import Demographics
+from elcid.episode_categories import InfectionService
+from intrahospital_api.apis.prod_api import ProdApi as ProdAPI
+from intrahospital_api.loader import create_rfh_patient_from_hospital_number
+
+from plugins.covid import lab, loader
+
+Q_GET_COVID_IDS = """
+SELECT DISTINCT Patient_Number FROM tQuest.Pathology_Result_view
+WHERE OBR_exam_code_Text = @test_name
+"""
+
+def pre_load_covid_patients():
```

Comment:
When the db is being updated this query will error out 

https://github.com/openhealthcare/elcid-rfh/blob/v0.39/intrahospital_api/apis/prod_api.py#L78

(I'm not sure if this is still an issue since they did their db update but it probably is)

Reply:
In practice this doesn't seem to be happening in the logs.
Even if it did, we effectively retry an hour later anyway so meh.